wallet/commands.go

```diff
@@ -685,6 +685,43 @@ func (wal *Wallet) SpendUnconfirmed(value bool) {
 	wal.multi.SaveUserConfigValue(dcrlibwallet.SpendUnconfirmedConfigKey, value)
 }
 
+// IsSpendUnconfirmed checks if start up password is set
+func (wal *Wallet) IsSpendUnconfirmed() bool {
+	return wal.multi.ReadBoolConfigValueForKey(dcrlibwallet.SpendUnconfirmedConfigKey, false)
+}
+
+func (wal *Wallet) BeepNewBlocks(value bool) {
+	wal.multi.SaveUserConfigValue(dcrlibwallet.BeepNewBlocksConfigKey, value)
+}
```

Comment:
The BeepNewBlocks, ConnectToPeer, UserAgent seem to call the same function, their difference being their config keys. You could just use one method "SaveConfig" and pass the config keys and update values from where they are called in the settings page.